recipes/lua/all/conanfile.py

```diff
@@ -1,72 +1,78 @@
-from conans import ConanFile, CMake, tools
 import os
 
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy, load, save, export_conandata_patches, apply_conandata_patches, collect_libs
+
+
 required_conan_version = ">=1.33.0"
```

Comment:
```suggestion
required_conan_version = ">=1.53.0"
```
self.settings.rm_safe requires Conan 1.53